I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/scripts/Chapter1/introScene.rpy", line 14, in script call
    call applyingForTheLab
  File "game/scripts/Chapter1/introScene.rpy", line 53, in script
    Friend "Applications? They world has fallen apart. 90% of humanity has been wiped out, and you want us to
TypeError: %o format: a number is required, not TagQuotingDict

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/scripts/Chapter1/introScene.rpy", line 14, in script call
    call applyingForTheLab
  File "game/scripts/Chapter1/introScene.rpy", line 53, in script
    Friend "Applications? They world has fallen apart. 90% of humanity has been wiped out, and you want us to
  File "C:\Users\Janelynn\Downloads\renpy-7.3.5-sdk\renpy\ast.py", line 708, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\Janelynn\Downloads\renpy-7.3.5-sdk\renpy\exports.py", line 1334, in say
    what = what % tag_quoting_dict
TypeError: %o format: a number is required, not TagQuotingDict

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Educational Visual Novel 1.0
Thu Feb 27 23:33:22 2020
